images/01-base/etc/waagent.conf

```diff
@@ -0,0 +1,81 @@
+# Windows Azure Linux Agent Configuration
```

Comment:
Which system service needs to use this `waagent.conf`?
The waagent service has its own `waegent.conf` file.